src/main/scala-2.11/org/apache/james/gatling/control/JamesWebAdministration.scala

```diff
@@ -45,18 +45,20 @@ class JamesWebAdministration(val baseUrl: URL) {
     new URL(s"$baseUrl/users/${username.value}/mailboxes")
 
   def createInbox(username: Username) = {
-    val mailboxesUrl = getMailboxesUrl(username)
-    wsClient.url(s"$mailboxesUrl/INBOX").put("")
+    createMailbox(username, "INBOX")
   }
 
   def createOutbox(username: Username) = {
-    val mailboxesUrl = getMailboxesUrl(username)
-    wsClient.url(s"$mailboxesUrl/Outbox").put("")
+    createMailbox(username, "Outbox")
   }
 
   def createSentBox(username: Username) = {
+    createMailbox(username, "Sent")
+  }
+
+  def createMailbox(username: Username, name: String) = {
```

Comment:
what about declaring two argument lists so that you can do partial application ?

Reply:
Like that?

Reply:
I was actually thinking of the opposite order so that you define the user out of the loop.